benchmark/cloud/template.1hr_benchmark.yml

```diff
@@ -110,3 +120,10 @@ options:
     plot_options:
       by_spc_cat: True
       by_hco_cat: True
+  #
+  # n_cores: Specify the number of cores to use:
+  # -1: Use all available cores
+  # -N: Use N cores
```

Comment:
Should this be `N` not `-N`? Same comment applies to rest of yml files modified here.

Reply:
That's what I thought but the manual for joblib.Parallel says:

>**Parameters**
>n_jobs: int, default: None
The maximum number of concurrently running jobs, such as the number of Python worker processes when backend=”multiprocessing” or the size of the thread-pool when backend=”threading”. If -1 all CPUs are used. If 1 is given, no parallel computing code is used at all, and the behavior amounts to a simple python for loop. This mode is not compatible with timeout. For n_jobs below -1, (n_cpus + 1 + n_jobs) are used. Thus for n_jobs = -2, all CPUs but one are used. None is a marker for ‘unset’ that will be interpreted as n_jobs=1 unless the call is performed under a [parallel_config()](https://joblib.readthedocs.io/en/stable/generated/joblib.parallel_config.html#joblib.parallel_config) context manager that sets another value for n_jobs.

In practice you would use -1 (all cores) or 1 (no cores).  I can update the YAML comments files accordingly

Reply:
In commit be766a2, all benchmark YAML files have been updated to use these comments, which are more accurate:
```yaml
  #
  # n_cores: Specify the number of cores to use.
  # -1: Use $OMP_NUM_THREADS         cores
  # -2: Use $OMP_NUM_THREADS - 1     cores
  # -N: Use $OMP_NUM_THREADS - (N-1) cores
  #  1: Disable parallelization (use a single core)
  #
  n_cores: -1
```